File: CS/MachineLearningDemo/Data/Feed.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using ml.Attributes;

namespace MachineLearningDemo.Data
{
    public class Feed
    {
        public string Source { get; set; }

        [StringFeature(SplitType=StringType.Word)]
        public string Content { get; set; }

        public int Cluster { get; set; }
    }
}

File: VB/MachineLearningDemo/Data/Feed.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports ml.Attributes

Namespace MachineLearningDemo.Data
    Public Class Feed
        Public Property Source() As String

        <StringFeature(SplitType:=StringType.Word)> _
        Public Property Content() As String

        Public Property Cluster() As Integer
    End Class
End Namespace
